queries.sql

```diff
@@ -9,3 +9,46 @@ SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+update animals set species = name where name like '%mon%'; 
+update animals set species = 'pokemon' where species is null;
+
+Begin transaction;
+delete from animals where date_of_birth > '2022-1-1';
+savepoint to one;
+update animals set weight_in_kg = weight_in_kg * -1;
+rollback task_one
+update animals set  weight_in_kg = weight_in_kg * 1 where weight_in_kg <0;
+commit transaction 
+
+
+select count (name) from animals;
+select * from animals where escape_attempts =0; 
+select avg(weight_in_kg) from animals;
+select name min(weight_in_kg) and max(weight_in_kg) from animals group by name;
+select neutered, max(escape_attempts) from animals group by neutered;
+select name, avg(escape_attempts) from animals where date_of_birth BETWEEN '1990-1-1' AND '2000-12-31' group by name;
+
```

Comment:
- [ ] Please we should write the query to update the animal name in the `data.sql`.